src/components/header/Header.css

```diff
@@ -240,7 +247,7 @@
   }
 
   .header__menu-overlay_type_active {
-    transition: opacity 01s ease;
+    transition: opacity 0.1s ease;
```

Comment:
I did make this 1s intentionally when I came through here a bit ago. I know that flash of overlay when the window shrinks is not ideal, I wasn't sure how to avoid that. The benefit of the 1s instead of .1s is that I think the opening of the mobile menu looks cleaner. Personally, I think it's a worthwhile sacrifice to keep it at 1s, as people who end up using the mobile menu probably aren't changing the window size anyways, and anyone on desktop proooobably won't ever go that small? In any case, that's just my opinion, I'll mark this as `change requested` so that it's flagged, but I'm fine with whatever, I just wanted to share my reasoning.

Reply:
I agree that a smooth transition trumps the overlay flash because opening the menu happens 100% of the time on mobile, and resizing the desktop is rare.

If you set the visibility to 1s and opacity 1s, the menu slides up with the same timing as the overlay so it doesn't look cut off.

Reply:
Yeah sorry, this was a mistake from my side! I thought it was a typo, because Jake set it to 01s; shouldn’t it be written just as 1s?